### PR TITLE
Adding codec constraints

### PR DESCRIPTION
**Pull request by charu0291-zz**
Adding page to specify different video codecs. 

To capture stats for different scenarios.

### CODE REVIEW

File: src/content/peerconnection/codec_constraints/js/test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*
```
Comment (MirkoBonadei) [inline src/content/peerconnection/codec_constraints/js/test.js:1]: Is this file needed?
It seems identical to https://github.com/webrtc/samples/blob/gh-pages/src/content/peerconnection/pc1/js/test.js.

File: src/content/peerconnection/codec_constraints/css/main.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
```
Comment (MirkoBonadei) [inline src/content/peerconnection/codec_constraints/css/main.css:2]: s/2015/2017

File: src/content/peerconnection/codec_constraints/index.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<!--
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
```
Comment (MirkoBonadei) [inline src/content/peerconnection/codec_constraints/index.html:3]: s/2015/2017

File: src/content/peerconnection/codec_constraints/js/main.js [hunk 4]
```diff
@@ -0,0 +1,403 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
```
Comment (MirkoBonadei) [inline src/content/peerconnection/codec_constraints/js/main.js:2]: s/2015/2017

**Review by MirkoBonadei — APPROVED** [review]
LGTM.

Can you use the squash & merge button and then add a description to the generated commit?

**Comment by fippo** [comment]
doing a rebase (git rebase -i master), squashing locally and then pushing (git push origin adding_codec_constraints --force-with-lease) followed by the normal merge will create less woes ;-)

File: src/content/peerconnection/codec_constraints/css/main.css (hunk 2 above)
Comment (Billiard0514) [reply src/content/peerconnection/codec_constraints/css/main.css:2]: Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
